src/app/lib/config/services/bengali.js

```diff
@@ -4,15 +4,33 @@ import {
   C_GHOST,
   C_POSTBOX_30,
 } from '@bbc/psammead-styles/colours';
-import { bengali } from '@bbc/gel-foundations/scripts';
+import { bengali, bengaliNotoSerif } from '@bbc/gel-foundations/scripts';
 import { bengali as brandSVG } from '@bbc/psammead-assets/svgs';
 import {
   F_SHONAR_BANGLA_BOLD,
   F_SHONAR_BANGLA_REGULAR,
+  F_NOTO_SERIF_BENGALI_BOLD,
```

Comment:
this might be a very confusing function name. It looks like a string name

Reply:
Yeah I found that strange as well